lib/utils/serializeGraphQLError.ts

```diff
@@ -0,0 +1,36 @@
+import { ErrorOpts } from "../errors";
+
+type GraphQLAPIError = {
+  response?: {
+    errors?: [
+      {
+        message?: string;
+        extensions?: {
+          status?: number;
+          type?: string;
+        };
+      }
+    ];
+  };
+};
+
+export const serializeGraphQLError = (graphQLError: GraphQLAPIError): ErrorOpts => {
+  const errorOptions: ErrorOpts = {};
+
+  const errorDetails =
```

Comment:
if https://github.com/kontist/js-sdk/pull/22 gets merged before this I can simplify this quite a bit 🙂

Reply:
done

Reply:
Thank you 🙏 I changed it here: badbc826c2493dedc1958e1833a47a95af89cb69